src/core/agent/services/signifyNotificationService.ts

```diff
@@ -480,6 +480,7 @@ class SignifyNotificationService extends AgentService {
               );
               if (connectionRecord) {
                 connectionRecord.pending = false;
+                connectionRecord.createdAt = new Date();
```

Comment:
Since the operation is done, this should be `operation.response.dt`

Reply:
Ah yes, I will update this